reference/content-types/text_line.rst

```diff
@@ -20,4 +28,7 @@ Example
         <meta>
             <title lang="en">Title</title>
         </meta>
+        <params>
+            <param name="headline" value="true"/>
```

Comment:
@marcelmoosbrugger Curious about that one two, that's what we have actually agreed on :wink: